dist/ci/testenv-leap151/run_as_tester

```diff
@@ -0,0 +1,7 @@
+#! /bin/sh
```

Comment:
Whitespace

Reply:
It's perfectly fine with it and I find it easier to read - but I won't argue here